packages/react/package.json

```diff
@@ -62,6 +62,8 @@
   },
   "devDependencies": {
     "@testing-library/react": "^11.0.4",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
This looks strange. What is the reason why we now need it here? Also, there is a `react` dependency inside the `peerDependencies` already